test-integration/helpers/student-dashboard.coffee

```diff
@@ -0,0 +1,100 @@
+_ = require 'underscore'
+CourseSelect = require './course-select'
+
+# A base class for a "Panel" type DOM object
+# If anyone finds this useful, feel free to expand and extract to helpers
+class PanelElement
+
+  constructor: (test, el) ->
+    @_test = test
+    if _.isString(el)
+      el = @test.driver.findElement(css: el)
```

Comment:
`PanelElement` is a base class that holds a reference to the element that defines itself.  The parent class will then use that element reference to locate sub-elements, `DashboardTask` below is a good example of how it's used.